libs/barista-components/drawer-table/src/drawer-table.scss

```diff
@@ -0,0 +1,18 @@
+$turquoise-600: #00a1b2;
```

Comment:
You can import our colorpalette from `@import '../../core/src/style/variables';`
It should not be necessary to redefine variables (especially color ones).